packages/http-client-python/generator/pygen/codegen/templates/model_dpg.py.jinja2

```diff
@@ -1,4 +1,5 @@
 {# actual template starts here #}
+import os
```

Comment:
Trigger pipeline to see whether CI fails if there is lint issue in generated SDK code.